docs/resources/integration_custom_auth_action.md

```diff
@@ -0,0 +1,87 @@
+---
+page_title: "genesyscloud_integration_custom_auth_action Resource - terraform-provider-genesyscloud"
+subcategory: ""
+description: |-
+  Genesys Cloud Integration Actions. See this page for detailed information on configuring Actions: https://help.mypurecloud.com/articles/add-configuration-custom-actions-integrations/
+---
+# genesyscloud_integration_custom_auth_action (Resource)
+
+Genesys Cloud Integration Actions. See this page for detailed information on configuring Actions: https://help.mypurecloud.com/articles/add-configuration-custom-actions-integrations/
+
+## API Usage
+The following Genesys Cloud APIs are used by this resource. Ensure your OAuth Client has been granted the necessary scopes and permissions to perform these operations:
+
+* [GET /api/v2/integrations/actions](https://developer.genesys.cloud/api/rest/v2/integrations/#get-api-v2-integrations-actions)
+* [GET /api/v2/integrations/actions/{actionId}](https://developer.genesys.cloud/api/rest/v2/integrations/#get-api-v2-integrations-actions--actionId-)
```

Comment:
I just remembered that since these are drafts, we need to include the draft endpoints in the list. There are draft analogues for all endpoints, but these should cover all of what we're modifying in this case: 
PATCH `/api/v2/integrations/actions/{actionId}/draft`, POST `/api/v2/integrations/actions/{actionId}/draft/publish`, GET `
/api/v2/integrations/actions/{actionId}/draft`
The permissions for drafts are the same as regular actions (D.A.V.E. permissions) so if they had permissions for the endpoints already listed, they'd also be covered.

Reply:
Added